origami/client.py

```diff
@@ -610,7 +610,51 @@ async def check_success(resp: GenericRTUReplySchema[TopicActionReplyData]):
             cell_id,
             properties=V2CellContentsProperties(source=contents),
         )
-        tracker = CellContentsDeltaReply.register_callback(self, req, check_success)
+        tracker = FileDeltaReply.register_callback(self, req, check_success)
+        await self.send_rtu_request(req)
+        return await asyncio.wait_for(tracker.next_trigger, timeout)
+
+    @_requires_ws_context
+    @_default_timeout_arg
+    async def delete_cell(self, file: NotebookFile, cell_id: str, timeout: float):
+        """Sends an RTU request to delete a particular cell in a particular file."""
+
+        async def check_success(resp: GenericRTUReplySchema[TopicActionReplyData]):
+            if not resp.data.success:
+                logger.error(f"Failed to submit cell delete for file {file.id} -> {cell_id}")
```

Comment:
Should we raise or just log here you think? It might make sense to simply crash

Reply:
Yeah we should crash here. Anything further is bound to fail anyway if this doesn't go through.